lib/AST/RawComment.cpp

```diff
@@ -130,60 +131,43 @@ static RawComment toRawComment(ASTContext &Context, CharSourceRange Range) {
   return Result;
 }
 
-RawComment Decl::getRawComment(bool SerializedOK) const {
-  if (!this->canHaveComment())
-    return RawComment();
-
-  // Check the cache in ASTContext.
-  auto &Context = getASTContext();
-  if (Optional<std::pair<RawComment, bool>> RC = Context.getRawComment(this)) {
-    auto P = RC.value();
-    if (!SerializedOK || P.second)
-      return P.first;
-  }
```

Comment:
Heh, just noticed this predicate was wrong anyway, and allowed us to retrieve a serialized entry from the cache even if `!SerializedOK`.